packages/extension-chakra-storefront/src/components/list-menu/list-menu-popover.jsx

```diff
@@ -14,45 +14,33 @@ import {ListMenuTrigger} from '../../components/list-menu/list-menu-trigger'
 
 const ListMenuPopover = ({contentComponent, item, name, itemsKey, maxColumns}) => {
     const [open, setOpen] = useState(false)
-    const onOpen = () => setOpen(true)
-    const onClose = () => setOpen(false)
+    const handleOpenChange = (details) => {
+        setOpen(details.open)
+    }
+
     const ContentComponent = contentComponent || ListMenuContent
     const recipe = useSlotRecipe({key: 'listMenu'})
     const styles = recipe()
 
     return (
-        <Box onMouseLeave={onClose}>
```

Comment:
We now removed the customization to open the menu on hover. Now preferring to rely on the default click behaviour of Chakra popover.